contrib/intel/jenkins/common.py

```diff
@@ -23,37 +23,29 @@ def run_command(command):
         sys.exit(p.returncode)
 
 
-Prov = collections.namedtuple('Prov', 'core util')
+Prov = collections.namedtuple('Prov', 'core enable force_dl')
```

Comment:
may be we could rename the core parameter of the tuple. core made more sense when the tuple had  the form (core, util).

Reply:
sure, 'name' makes more sense.